addons/mass_mailing_sms/models/mailing_list.py

```diff
@@ -63,3 +63,106 @@ def _mailing_get_opt_out_list_sms(self, mailing):
         opt_out_contacts = subscriptions.filtered(lambda sub: sub.opt_out).mapped('contact_id')
         opt_in_contacts = subscriptions.filtered(lambda sub: not sub.opt_out).mapped('contact_id')
         return list(set(c.id for c in opt_out_contacts if c not in opt_in_contacts))
+
+    def action_merge(self, src_lists):
+        """
+            Insert all the contact from the mailing lists 'src_lists' to the
+            mailing list in 'self'. Delete mailing lists 'src_lists' after
+            the merge except the destination mailing list 'self'.notes
+        """
+        # Explation of the SQL query with an example. There are the following lists
```

Comment:
Note that if we achieved a "clean" override, we would just have to explain how adding mobile support changes mailing list merge, instead of rewriting code and docstring :) .